### DISCUSSION TITLE
GitHub issue label for value classes

### DISCUSSION THREAD

**Discussion by aSemy** [mockk/mockk#858] [Ideas]
Hi, I wanted to pick up some of the value class tickets now the ball is rolling #849 

To help with this, can you please add a label to the value class tickets? This would help me to easily find the relevant issues. Thanks!

https://github.com/mockk/mockk/issues?q=%22value+class%22+

**Comment by Raibaz** [comment]
Good idea, thanks!

I created a "value classes" label and tagged most of the issues i could find about them.

**Reply by aSemy** [reply]
Thanks!

There are some more here: https://github.com/mockk/mockk/issues?q=%22jvminline%22+OR+%22value+class%22+OR+%22inline+class%22

**Reply by Raibaz** [reply]
Done :)

**Reply by aSemy** [reply]
More again here :) https://github.com/mockk/mockk/issues?q=%22value+classes%22
